engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/meta/EqualsComplexNode.java

```diff
@@ -39,24 +39,26 @@ public static EqualsComplexNode build() {
     return EqualsComplexNodeGen.create();
   }
 
-  public abstract boolean execute(@AcceptsError Object left, @AcceptsError Object right);
```

Comment:
Why was the `@AcceptsError` removed here?

Reply:
`@AcceptError` is Enso Builtins DSL, but this node is no longer processed by the DSL. `EqualsNode` is and it does contain these annotations.